### PR TITLE
Repo clean up

### PR DESCRIPTION
**Pull request by yizshi**
- Add helper function since java 17 don't allow mock random 
- update okhttp version for known vulnerabilities 
- update example source target allow java 17 builds.

Tested locally with both java 8 and java 17, both pass unit test and able to run admin example

### CODE REVIEW

File: duo-client/pom.xml [hunk 1]
```diff
@@ -44,7 +44,7 @@
     <dependency>
       <groupId>com.squareup.okhttp3</groupId>
       <artifactId>okhttp</artifactId>
-      <version>4.9.1</version>
+      <version>4.10.0</version>
```
Comment (yizshi) [inline duo-client/pom.xml:47]: https://security.snyk.io/vuln/SNYK-JAVA-COMSQUAREUPOKHTTP3-2958044
It recommend to upgrade to 4.9.2 or higher. So I just bump to latest release. I can upgrade to 4.9.2 if it is too risky to jump to the latest

File: duo-client/src/test/java/com/duosecurity/client/HttpRateLimitRetryTest.java [hunk 2]
```diff
@@ -22,8 +22,6 @@
 public class HttpRateLimitRetryTest {
     @Mock
     private OkHttpClient httpClient;
-    @Mock
-    private Random random;
```
Comment (yizshi) [inline duo-client/src/test/java/com/duosecurity/client/HttpRateLimitRetryTest.java:26]:
```
org.mockito.exceptions.base.MockitoException: 
Mockito cannot mock this class: class java.util.Random.
```
This was the problem, and quick google search lead me to know java 17 don't allow mock to tap random method anymore. The next easiest thing I can think of is to create a helper method so we can just mock the helper method.

File: duo-example-admin/pom.xml [hunk 3]
```diff
@@ -63,8 +63,8 @@
         <artifactId>maven-compiler-plugin</artifactId>
         <version>2.3.2</version>
         <configuration>
-          <source>1.6</source>
-          <target>1.6</target>
+          <source>1.8</source>
```
Comment (yizshi) [inline duo-example-admin/pom.xml:66]: 1.8 shouldn't be backwards incompatible, and bring the example to the same target as the client itself.

**Review by axorna — APPROVED** [review]

File: duo-client/src/test/java/com/duosecurity/client/HttpRateLimitRetryTest.java (hunk 2 above)
Comment (axorna) [reply duo-client/src/test/java/com/duosecurity/client/HttpRateLimitRetryTest.java:26]: I think this is fine, https://security.snyk.io/package/maven/com.squareup.okhttp3:okhttp/4.10.0 says there are no vulnerabilities and it was released june of last year. They're already working on 5.0.0-alpha versions, so I think 4.10.0 is safe.

File: duo-example-admin/pom.xml (hunk 3 above)
Comment (axorna) [reply duo-example-admin/pom.xml:66]: lgtm!

File: duo-client/src/test/java/com/duosecurity/client/HttpRateLimitRetryTest.java (hunk 2 above)
Comment (axorna) [reply duo-client/src/test/java/com/duosecurity/client/HttpRateLimitRetryTest.java:26]: lgtm!

File: duo-client/pom.xml (hunk 1 above)
Comment (axorna) [reply duo-client/pom.xml:47]: I think this is fine, https://security.snyk.io/package/maven/com.squareup.okhttp3:okhttp/4.10.0 says there are no vulnerabilities and it was released june of last year. They're already working on 5.0.0-alpha versions, so I think 4.10.0 is safe.

**Pull request merged**